packages/react-core/src/components/InputGroup/examples/InputGroupWithDropdown.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Button, InputGroup, TextInput, Dropdown, DropdownToggle, DropdownItem } from '@patternfly/react-core';
+
+export const InputGroupWithDropdown: React.FunctionComponent = () => {
+  const [isOpen, setIsOpen] = React.useState<boolean>(false);
+  const [selected, setSelected] = React.useState<string>('');
+
+  const onToggle = isOpen => {
+    setIsOpen(isOpen);
+  };
+
+  const onSelect = event => {
+    setIsOpen(false);
+    setSelected(event.currentTarget.value);
+  };
```

Comment:
I'm wondering if we should keep the event param (and likewise the `selected` state) in this example. I know other examples that utilize a dropdown only use the `onSelect` prop to essentially close the dropdown menu (the design guidelines also link to the Select component for selecting from a list of options).

@tlabaj what do you think?

Reply:
I agree that there is no need to track the selected state since this is a dropdown and not a select.  You can have the `onSelect` toggle the dropdown closed like we do in our examples for the `Dropdown` component.

Reply:
Okay, I removed the event and selected state